src/functions/forEachObjectArray.js

```diff
@@ -24,16 +24,14 @@ module.exports = async (d) => {
 
     let el = {};
     let i = 0;
-    for (const key in d.data.objects[name]) {
-        if (d.data.objects[name].hasOwnProperty(key)) {
+    for (const key in d.data.objects[name][property]) {
+        if (d.data.objects[name][property].hasOwnProperty(key)) {
```

Comment:
need a check before this for loop to see if d.data.objects[name][property] is an iterable else it will error out